src/commands/deploy/utils.ts

```diff
@@ -716,3 +723,31 @@ export async function uploadEnvVarsFromFile(
         );
     }
 }
+
+// Upload the project code to S3 for in-browser editing
+export async function uploadUserCode(name: string, region: string, stage: string): Promise<void> {
+    const tmpFolderProject = await createTemporaryFolder();
+    debugLogger.debug(`Creating archive of the project in ${tmpFolderProject}`);
+    const promiseZip = zipDirectory(
+        process.cwd(),
+        path.join(tmpFolderProject, "projectCode.zip"),
+        false,
+        [
+            "**/node_modules/*",
+            "./node_modules/*",
+            "node_modules/*",
+            "**/node_modules",
+            "./node_modules",
+            "node_modules",
+            "node_modules/**",
+            "**/node_modules/**",
+        ],
```

Comment:
It seems to me that the exclusion list is smaller than it was before refactoring.
I believe we are adding the `.git` directory and any other temporary directories used by vercel, opennnext in the zip.

Is this something we want?

I think we should at least keep ignoring the `.git` related stuff.

Reply:
All dot files and folders are ignored. That's why I did the change with `zipDirectory`.